public/pyscript/main.py

```
@@ -102,0 +1,114 @@
+from PIL import Image
+from io import BytesIO
+
+from pyscript import when # noqa # pyright: ignore [reportAttributeAccessIssue]
```

Ruff (0.11.9)

Unused blanket `noqa` directive

Remove unused `noqa` directive

(RUF100)